databind/src/main/java/gov/nist/secauto/metaschema/databind/io/yaml/YamlOperations.java

```diff
@@ -84,9 +84,9 @@ private YamlOperations() {
    */
   @SuppressWarnings({ "unchecked", "null" })
   @NonNull
-  public static Map<String, Object> parseYaml(Path target) throws IOException {
-    try (BufferedReader reader = Files.newBufferedReader(target.toAbsolutePath(), StandardCharsets.UTF_8)) {
-      return (Map<String, Object>) YAML_PARSER.load(reader);
+  public static Map<String, Object> parseYaml(URI target) throws IOException {
+    try (InputStream is = ObjectUtils.notNull(target.toURL().openStream())) {
```

Comment:
Per our conversation on the call, you would prefer this @david-waltermire?

```suggestion
try (BufferedInputStream is = new BufferedInputStream(ObjectUtils.notNull(target.toURL().openStream()))) {
```

Reply:
Changed locally and pushing up with pairing changes momentarily.

Reply:
Adjusted in https://github.com/usnistgov/metaschema-java/pull/298/commits/9f5fff5b230f5a08284469acd277fb4ff1b26a9d.